crates/router/src/core/payments/helpers.rs

```diff
@@ -707,37 +707,6 @@ pub async fn make_pm_data<'a, F: Clone, R>(
                     }
                     (_, _) => pm,
                 }
-            } else if payment_method_type == Some(storage_enums::PaymentMethodType::Wallet) {
```

Comment:
Wait, why were these lines removed? They were recently added by @Sangamesh26 in #388.

Reply:
Sorry, my bad, removed this accidentally while merging